File: week3-app-development/javascript/Ex6a_Create_Table.js
const { client } = require("./connection");

console.log("========================================");
console.log("Start exercise");

(async () => {
  try {
    await client.execute(`
        CREATE TABLE todoitems (
            user_id         TEXT,
            item_id         TIMEUUID,
            title           TEXT,
            completed       BOOLEAN,
            PRIMARY KEY ((user_id), item_id)
        ) WITH CLUSTERING ORDER BY (item_id ASC);`);
    console.log("SUCCESS");
  } catch (e) {
    console.log(e);
  }
  console.log("========================================");
  process.exit();
})();

File: week3-app-development/python/Ex6a_Create_Table.py
#!/usr/bin/env python3
from connection import session

print('========================================')
print('Start exercise')
try:
    session.execute("""CREATE TABLE IF NOT EXISTS todoitems (
                        user_id         TEXT,
                        item_id         TIMEUUID,
                        title           TEXT,
                        completed       BOOLEAN,
                        PRIMARY KEY ((user_id), item_id)
                    ) WITH CLUSTERING ORDER BY (item_id ASC);
                    """)
except Exception as e:
    print(e)
    print('Failure')
else:
    print('Success')
print('========================================')
